gns3server/modules/qemu/__init__.py

```diff
@@ -39,6 +40,27 @@ class Qemu(BaseManager):
     _VM_CLASS = QemuVM
 
     @staticmethod
+    def get_kvm_archs():
+        """
+        Gets a list of architectures for which KVM is available on this server.
+
+        :returns: List of architectures for which KVM is available on this server.
+        """
+        kvm = []
+        x86_64_aliases = ["x86_64", "x86-64", "x64", "AMD64", "amd64", "Intel 64", "EM64T"]
```

Comment:
Due to the fact we control the input files via the registry we can be strict and keep only one name by architecture I think. I don't see problem with what you have propose but I fear to have trouble in the future.